### DISCUSSION TITLE
Codemirror-Autocompletion: <currentTiddler>?

### DISCUSSION THREAD

**Discussion by rimi** [TiddlyWiki/TiddlyWiki5#6737] [Q&A] [answered]
Hi all,

i'm currently working an some custom code-completion behaviour in CodeMirror and was wondering if it is possible to find out the name of the tiddler currently working on (aka &lt;currentTiddler&gt;) inside of my hint-function (imagine my "working-area" like `$:/plugins/tiddlywiki/codemirror/addon/hint/anyword-hint.js`). Anyone did that before?

Thanks!
Mirko

**Comment by saqimtiaz** [comment, marked as answer]
It has been a long time since I worked with CodeMirror but try this:  `editor.options.editWidget.getVariable("currentTiddler")`

**Reply by rimi** [reply]
Thanks for that, it works for me!
